lib/base.css

```diff
@@ -80,3 +82,37 @@ ul {
 [tabindex="-1"]:focus {
   outline: none !important;
 }
+
+/**
+ * Remove the outline on focused links when they are also active or hovered
+ * in all browsers.
+ */
+
+a:active,
+a:hover {
+  outline-width: 0;
+}
+
+/**
+ * Change the font styles in all browsers.
+ */
+
+button,
+input,
+optgroup,
+select,
+textarea {
+  font-family: sans-serif;
```

Comment:
This is just copied straight from normalize. They were the opinionated rules removed in `6.0.0`. There is a bit of contention over whether that should have happened so in the meantime I think they can live in `base`.